requirements.txt

```diff
@@ -4,6 +4,7 @@ biopython # Enables Pubmed widget.
 conllu
 docx2txt>=0.6
 gensim>=4.1.2  # https://github.com/RaRe-Technologies/gensim/issues/3226
+httpx
```

Comment:
We are actually not adding a new dependency in the environment since Orange already depends on HTTPX.